subsys/fw_info/Kconfig

```diff
@@ -75,6 +75,8 @@ config FW_INFO_HARDWARE_ID
 	default 51 if SOC_SERIES_NRF51X
 	default 52 if SOC_SERIES_NRF52X
 	default 53 if SOC_SERIES_NRF53X
+	default 154 if SOC_SERIES_NRF54LX
```

Comment:
how you get these numbers?

Reply:
Going with established scheme meets the wall here. We have two 54's. Any better ideas are more than welcome.

Reply:
You can go with 54 for NRF54LX
For 54H this won't be supported